README.md

```diff
@@ -1,2 +1,309 @@
+    --bucket $BUCKET \
+    --region $REGION \
+    --create-bucket-configuration LocationConstraint=$REGION
+```
+NOTE: us-east-1 does not support a `LocationConstraint`.  If your region is `us-east-1`, omit the bucket configuration:
+
+```bash
+aws s3api create-bucket \
+    --bucket $BUCKET \
+    --region us-east-1
+```
+
+## Set permissions for Velero
+
+### Set permissions with an IAM user
+
+For more information, see [the AWS documentation on IAM users][10].
+
+1. Create the IAM user:
+
+    ```bash
+    aws iam create-user --user-name velero
+    ```
+
+    If you'll be using Velero to backup multiple clusters with multiple S3 buckets, it may be desirable to create a unique username per cluster rather than the default `velero`.
+
+2. Attach policies to give `velero` the necessary permissions:
+
+    ```
+    cat > velero-policy.json <<EOF
+    {
+        "Version": "2012-10-17",
+        "Statement": [
+            {
+                "Effect": "Allow",
+                "Action": [
+                    "ec2:DescribeVolumes",
+                    "ec2:DescribeSnapshots",
+                    "ec2:CreateTags",
+                    "ec2:CreateVolume",
+                    "ec2:CreateSnapshot",
+                    "ec2:DeleteSnapshot"
+                ],
+                "Resource": "*"
+            },
+            {
+                "Effect": "Allow",
+                "Action": [
+                    "s3:GetObject",
+                    "s3:DeleteObject",
+                    "s3:PutObject",
+                    "s3:AbortMultipartUpload",
+                    "s3:ListMultipartUploadParts"
+                ],
+                "Resource": [
+                    "arn:aws:s3:::${BUCKET}/*"
+                ]
+            },
+            {
+                "Effect": "Allow",
+                "Action": [
+                    "s3:ListBucket"
+                ],
+                "Resource": [
+                    "arn:aws:s3:::${BUCKET}"
+                ]
+            }
+        ]
+    }
+    EOF
+    ```
+    ```bash
+    aws iam put-user-policy \
+      --user-name velero \
+      --policy-name velero \
+      --policy-document file://velero-policy.json
+    ```
+
+3. Create an access key for the user:
+
+    ```bash
+    aws iam create-access-key --user-name velero
+    ```
+
+    The result should look like:
+
+    ```json
+    {
+      "AccessKey": {
+            "UserName": "velero",
+            "Status": "Active",
+            "CreateDate": "2017-07-31T22:24:41.576Z",
+            "SecretAccessKey": <AWS_SECRET_ACCESS_KEY>,
+            "AccessKeyId": <AWS_ACCESS_KEY_ID>
+      }
+    }
+    ```
+
+4. Create a Velero-specific credentials file (`credentials-velero`) in your local directory:
+
+    ```bash
+    [default]
+    aws_access_key_id=<AWS_ACCESS_KEY_ID>
+    aws_secret_access_key=<AWS_SECRET_ACCESS_KEY>
+    ```
+
+    where the access key id and secret are the values returned from the `create-access-key` request.
+
```

Comment:
It looks like the actual `velero install` step got dropped here, i.e. https://github.com/vmware-tanzu/velero/blob/master/site/docs/v1.1.0/aws-config.md#install-and-start-velero.

Reply:
it's on ln 244.

Reply:
Ah, I see. ok 👍